apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -128,6 +119,20 @@ export function ModuleFunction({
                         )}
                     />
                 </div>
+                {execute.error || execute.data ? (
+                    <FunctionExecutionResult
+                        error={
+                            execute.error
+                                ? (execute.error as Error).message || 'Error'
+                                : false
+                        }
+                        result={execute.data || null}
+                        onClear={() => {
+                            execute.reset();
+                            reset();
```

Comment:
I actually don't think this should reset the form. I might want to run something multiple times with the same inputs, and it feels unintuitive that dismissing the result would fully reset the form.